documentation/specs/RadioGroup.md

```diff
@@ -42,9 +42,6 @@ type RadioGroupProps = {
   /** Whether the radio is disabled. */
   isDisabled?: boolean;
 
-  /** Whether or not the radio group label is visually hidden. Only recommended for when the grouping is visually implicit. */
-  isLabelVisuallyHidden?: boolean;
```

Comment:
**question** does it not make sense for this component to be able to visually hide a label?

Reply:
this is a good question and became slightly more complicated when i implemented the component. yes it does make sense to allow for visually hiding the label. however react aria has a built-in mechanism for allowing the user to provide a custom aria label either through `aria-label`, `aria-labelledby`, `aria-describedby`, and/or `aria-details`. in fact, react aria writes a console warning describing what to do when a label is not passed. with that built-in mechanism, instead of using `isLabelVisuallyHidden` which would limit us to assigning an `aria-label` on the input, i thought it'd be best to support the react aria way of doing it, allowing for `label` to be optional, but in which case the consumer would need to pass one of the labeling props. then the consumer could use `aria-labelledby`, as an example, if they have a more complex use case beyond just an `aria-label`.

i actually added to this type the `AriaLabelingProps` from `react-aria` that supports the props mentioned above.

let me know if that makes sense